assets/src/scripts/monitoring-location/components/map/legend.js

```diff
@@ -34,30 +26,50 @@ const fetchLayerLegend = function(layer, defaultName) {
         });
 };
 
-export const addMonitoringLocationMarker = function(legendControl) {
-        const siteLegendList = select(legendControl.getLegendListContainer()).append('ul')
+/*
+ * Draws a monitoring location marker within the legendListContainer
+ * @param {D3 selection} legendListContainer
+ */
+export const drawMonitoringLocationMarkerLegend = function(legendListContainer) {
```

Comment:
I changed the methods here to take the legendListContainer so that they are not dependent on Leaflet at all. This makes it easier to test the functions and it follows the pattern for most of our D3 rendering code.

Reply:
Much nicer 👍